src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -834,6 +884,38 @@ void Pane::SplitHorizontal(const GUID& profile, const TermControl& control)
     _Split(SplitState::Horizontal, profile, control);
 }
 
+// Method Description:
+// - Determines whether the pane can be split.
+// Arguments:
+// - splitType: what type of split we want to create.
+// Return Value:
+// - True if the pane can be split. False otherwise.
+bool Pane::_CanSplit(SplitState splitType)
+{
+    const bool changeWidth = _splitState == SplitState::Vertical;
+
+    const Size actualSize{ gsl::narrow_cast<float>(_root.ActualWidth()),
+                           gsl::narrow_cast<float>(_root.ActualHeight()) };
+

```

Comment:
This might need to account for the separator size - when we split the pane, each child is not going to half the parent, it's going to have (half the parent - the size of the separator)